index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] The container of this section should be a `<header>` tag. Please kindly fix that as this makes your html more [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp)